index.bs

```diff
@@ -243,6 +243,50 @@ scenarios. Additional scenarios, including sample code, are given later in [[#sa
     * Web page shows that the selected user is signed in, and navigates to the signed-in page.
 
 
+### New device enrollment ### {#usecase-new-device-enrollment}
```

Comment:
I'm thinking we ought to use the term "registration" rather than enrollment here because "enrollment" is what one does for their biometric authz gestures.

Reply:
Ok!